packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -274,6 +274,15 @@ class Scrollable extends StatefulWidget {
   /// Determined by the [axisDirection].
   Axis get axis => axisDirectionToAxis(axisDirection);
 
+  String get _selectionSeparator {
```

Comment:
This should probably be set in _ScrollableSelectionContainerDelegate instead

Reply:
Not exactly sure what you want here. We need access to the axis to decide what separator to use, and _ScrollableSelectionContainerDelegate doesn't have access to that.

Reply:
I moved this logic to where we create the _ScrollableSelectionHandler. Let me know if you want any changes.